deli_counter.rb

```diff
@@ -1 +1,27 @@
-# Write your code here.
\ No newline at end of file
+# Write your code here.
+
+def line(katz_deli)
+  if katz_deli.length == 0
```

Comment:
I wrote the tutorial mirroring the solution (because the solution and tutorial are on the same branch). Should I change the solution and Tutorial here?